### PR TITLE
replaced magic numbers :D

### PR DESCRIPTION
**Pull request by Rimuruweeb**
* Resolves: # 
* Target version: master 

### Summary
on line 603 couldn't replace 99 , idk was giving some error , i checked every number tho :P.

### CODE REVIEW

**Comment by Rash419** [comment]
@Rimuruweeb thanks for the PR 
DCO and Change-Id checks are failing if you click on the details you will find out what is the problem 
commit message should be meaningfull 
checkout this guide for more info: https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/

**Comment by pedropintosilva** [comment]
Thanks and welcome @Rimuruweeb ! As @Rash419 already pointed out you need to:

File: browser/src/map/handler/Map.Keyboard.js [hunk 1]
```diff
@@ -140,18 +140,18 @@ L.Map.Keyboard = L.Handler.extend({
 	},
 
 	keyCodes: {
-		pageUp:   33,
-		pageDown: 34,
-		enter:    13
+		pageUp:   UNOKey.PAGEUP,
```
[the remark this answers is not in the document]
Comment (Rimuruweeb) [reply browser/src/map/handler/Map.Keyboard.js:143]: Do you want me to revert backchanges in keycodes ? and everything expect keycodes are alright?

[the remark this answers is not in the document]
Comment (Rimuruweeb) [reply browser/src/map/handler/Map.Keyboard.js:143]: i mean your example gave this values  112 to UNOKey.F1.

[the remark this answers is not in the document]
Comment (Rimuruweeb) [reply browser/src/map/handler/Map.Keyboard.js:143]: sir thank you for your response but it says keycodes are not defined,
image

[the remark this answers is not in the document]
Comment (pedropintosilva) [reply browser/src/map/handler/Map.Keyboard.js:143]: @Rimuruweeb it is defined in https://github.com/CollaboraOnline/online/blob/master/browser/src/map/handler/Map.Keyboard.js#L320 
```js
var keyCode = ev.keyCode;
```

[the remark this answers is not in the document]
Comment (Rimuruweeb) [reply browser/src/map/handler/Map.Keyboard.js:143]: i think sir @eszkadev meant keyCodes not keyCode .

[the remark this answers is not in the document]
Comment (eszkadev) [reply browser/src/map/handler/Map.Keyboard.js:143]: @pedropintosilva no it is not what we want here

[the remark this answers is not in the document]
Comment (eszkadev) [reply browser/src/map/handler/Map.Keyboard.js:143]: @Rimuruweeb you need to add new names/memebers into that dictionary we are commenting right now.

https://github.com/CollaboraOnline/online/pull/4961/files/4be9181d18921604e4abcffba6f70472af52cc82#diff-5c4df738e4e71c87f575a954037607d6ffac23b63686d7de65dd029bca95eed6L142

that is good place where we can define buinding between   name  and number value

[the remark this answers is not in the document]
Comment (eszkadev) [reply browser/src/map/handler/Map.Keyboard.js:143]: > @eszkadev you want me to add name and numbers to this scope?
> 
> https://github.com/CollaboraOnline/online/blob/4be9181d18921604e4abcffba6f70472af52cc82/browser/src/map/handler/Map.Keyboard.js#L142

Yes

File: browser/src/map/handler/Map.Keyboard.js [hunk 2]
```diff
@@ -136,15 +138,89 @@ L.Map.Keyboard = L.Handler.extend({
 		39  : true, // right arrow
 		40  : true, // down arrow
 		45  : true, // insert
-		113 : true  // f2
+		113 : true  // f2sss
```
Comment (eszkadev) [inline browser/src/map/handler/Map.Keyboard.js:141]: this is probably mistake :) please revet this change

File: browser/src/map/handler/Map.Keyboard.js [hunk 3]
```diff
@@ -136,15 +138,89 @@ L.Map.Keyboard = L.Handler.extend({
...
 	},
 
+
 	keyCodes: {
-		pageUp:   33,
-		pageDown: 34,
-		enter:    13
+		BACKSPACE   : UNOKey.BACKSPACE,
```
Comment (eszkadev) [inline browser/src/map/handler/Map.Keyboard.js:146]: Please DO NOT USE `UNOKey` enum here. I said it was misleading comment from my side to show that dictionary as an example.

What we want here is to bind NAME with NUMBER. `pageUp: 33` and others should stay here, they are correct bindings. Please only add new ones for missing keyCodes used in this file.

like on the bottom we have `45`, you should add here: `INSERT: 45,`, then replace occurrences of `45` with `this.keyCodes.INSERT`  - this is done great already :)

**Review by eszkadev — CHANGES_REQUESTED** [review]

File: browser/src/map/handler/Map.Keyboard.js (hunk 3 above)
Comment (eszkadev) [reply browser/src/map/handler/Map.Keyboard.js:146]: ( `UNOKey` stores values like `1034` which are different than keyCodes used by the browser, `UNOKey` enum is for our server which uses different coding )

**Comment by eszkadev** [comment]
> Hello, is this PR fine now? Can we rebase and review again? Looks like a good improvement :)

No, it uses incorrect values currently.

**Comment by eszkadev** [comment]
Issue fixed with other commit.

**Pull request closed without merging**